src/amber/cli/templates/app/src/assets/javascripts/main.js

```diff
@@ -1 +1,23 @@
 import Amber from 'amber'
+
+if (!Date.prototype.toGranite) {
+  (function() {
+
+    function pad(number) {
+      if (number < 10) {
+        return '0' + number;
+      }
+      return number;
+    }
```

Comment:
thank you for not including `left_pad` for this ;-)

Reply:
copied it from the polyfill for toISOString() at MDN - always try to follow the experts when I can :-)